### ISSUE TITLE
installation with pip problem

### ISSUE THREAD

**Issue by davidedelvento** [pysal/pysal#1070]
This is related to issu #693 

- Platform information (I think it's irrelevant for this issue)
```
posix linux
```
- Python version (I think it's irrelevant for this issue)
```
3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609]
```
- SciPy version (I think it's irrelevant for this issue)
```
1.2.0
```
- NumPy version (I think it's irrelevant for this issue)
```
1.15.4
```

On my machine I have both python2 and python3 with the corresponding pip2 and pip3.
If I install with pip2 first, when subsequently installing with pip3 instead of downloading the appropriate python3 version for installing pip3 it installs the version previously cached by pip2. This should not happen (TM) and as far as I know it does not happen with any of the several dozens of other packages which I install both for python2 and python3

**Comment by sjsrey** [comment]
Thank you for reporting this.

Did the suggested fix in #693 resolve this for you?

After PySAL 2.0 is released (in a few weeks), it will be Python 3+ only.  Please see the [discussion of these changes](https://pysal.org/about.html) for further information and guidance for users who are on Python2 and want to use PySAL.

**Comment by davidedelvento** [comment]
@sjsrey Thank you very much for the quick reply. I am not an user of PySAL, but an admin of a number of servers where I automatically install it (via ansible) for the actual users. A manual test of the reinstall suggested in #693 fixed the issue for that server where I performed it, but I had not yet implemented it more broadly. Given the Python 3+ only status you mention I will not and I will point my users to your link. As far as I am concerned, you may close this issue. Thanks again.
